packages/botonic-dx/baseline/tsconfig.tests.json

```diff
@@ -0,0 +1,19 @@
+// Only used for verifying the code (& by jest)
+// It uses noEmit to have faster builds (up to 30%)
+{
+  "extends": "./tsconfig.json",
+  // include: [] makes jest faster,
+  // but it needs to be set for tsc to find the files to verify them. See https://www.notion.so/Typescript-performance-7aa77701970d4d49b8399efcee1b0754
+  "include": ["**/*.[jt]s", "**/*.[jt]sx", "../src/**/*.[jt]s", "../src/**/*.[jt]sx"],
+  "compilerOptions": {
+    "rootDir": "../",
+    // Options below are for compiling @botonic js/jsx files
+    "declaration": false,
+    "declarationDir": null,
+    "noEmit": true,
+    "types": [
+      "@types/jest"
```

Comment:
`@types/node` should be added here too.

Reply:
done. I removed it from sample-config to simplify